build-system/compile/generate/bento.js

```diff
@@ -13,19 +13,15 @@ const {getSharedBentoSymbols} = require('./shared-bento-symbols');
  * @param {Object<string, string[]>} packageSymbols
  * @return {string}
  */
-function generateBentoRuntimeEntrypoint(
-  packageSymbols = getSharedBentoSymbols()
-) {
+function generateBentoRuntimeEntrypoint(packageSymbols) {
+  packageSymbols = packageSymbols || getSharedBentoSymbols();
```

Comment:
I'm not sure I understand the purpose of this change. Can you explain how this is different? Note that the parameter is no longer treated as optional thus the type checking failure.

Reply:
it was to address someone else finding it odd that we were invoking functions in default arguments: https://github.com/ampproject/amphtml/pull/37386#discussion_r798008041

Reply:
re: type checking, I can change the jsdoc to indicate that it's now optional since we handle when it isn't passed in

Reply:
updated

Reply:
Indeed this was me finding the default-to-function-call uncommon enough to prefer doing it explicitly (albeit with the optional param update in JSDoc). Totally style preference, I don't feel strongly